native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -641,16 +642,20 @@ static proxy_balancer *add_balancer_node(nodeinfo_t *node, proxy_server_conf *co
     if (balancer && balancer->workers->nelts == 0) {
         /* Logic to copy the shared memory information to the balancer */
         balancerinfo_t *balan = read_balancer_name(&balancer->s->name[11], pool);
-        if (balan == NULL)
+        if (balan == NULL) {
             return balancer; /* Done broken */
+        }
         /* StickySession, StickySessionRemove we hack it via the lbpname (16 bytes) */
-        if (!balan->StickySession)
+        if (!balan->StickySession) {
             strcpy(balancer->s->lbpname, MC_NOT_STICKY);
-        else
+        }
+        else {
```

Comment:
Hm, does the clang formatter require the `else {` to be on a new line?

I would expect the 'else' to be on the same line as the bracket of the previous block, i.e.:

`} else {`

That's the style we have always used for Java - [example](https://github.com/modcluster/mod_cluster/blob/5babe75797db30c334f099f42992db130b784108/container/tomcat-8.5/src/main/java/org/jboss/modcluster/container/tomcat/TomcatConnector.java#L74https://github.com/modcluster/mod_cluster/blob/5babe75797db30c334f099f42992db130b784108/container/tomcat-8.5/src/main/java/org/jboss/modcluster/container/tomcat/TomcatConnector.java#L74). Starting a new line with `else` hurts my brain.

Reply:
Yes, it's how they do it in httpd, see https://github.com/modcluster/mod_proxy_cluster/pull/74 and related https://issues.redhat.com/browse/MODCLUSTER-783.

In clang-format, it's the BraceWrapping option. (I hate it too. :smiley:)

I guess we can change it, but the decision is on @jfclere I guess. We can either stick to httpd style, or go with the mod_cluster/Java part style.

Reply:
We can always arrange the style later if one day we give the mod_proxy_cluster the Apache httpd.
So we could with the mod_cluster/Java part style for the moment.

Reply:
I changed the style in the latest commit.